.bazelrc

```diff
@@ -2,6 +2,9 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
+# import site-specific configuration
```

Comment:
Nit: capitalize "import" in the comment so it doesn't look like a commented-out `import` line.

Reply:
Done.